test/python/circuit/test_extensions_standard.py

```diff
@@ -851,6 +852,27 @@ def test_z_reg_inv(self):
         self.assertEqual(instruction_set[1].qubits, (self.qr[1],))
         self.assertEqual(instruction_set[2].operation.params, [])
 
+    def test_global_phase(self):
+        qc = self.circuit
+        qc.append(GlobalPhaseGate(0.1), [])
+        self.assertEqual(self.circuit[0].operation.name, "global_phase")
+        self.assertEqual(self.circuit[0].operation.params, [0.1])
+        self.assertEqual(self.circuit[0].qubits, ())
+
+    def test_global_phase_inv(self):
+        instruction_set = self.circuit.append(GlobalPhaseGate(0.1), []).inverse()
+        self.assertEqual(len(instruction_set), 1)
+        self.assertEqual(instruction_set[0].operation.params, [-0.1])
+
+    def test_global_phase_matrix(self):  # , theta: float, expected: np.ndarray
+        """Test global_phase matrix."""
+        theta = 0.1
+        np.testing.assert_allclose(
+            GlobalPhaseGate(theta).__array__(),
+            np.array([[np.exp(1j * theta)]], dtype=complex),
+            atol=1e-7,
+        )
```

Comment:
Add a test to check consistency with the `QuantumCircuit.global_phase` attribute. Do this by creating two circuits, one that has a GlobalPhaseGate, and the other that is initialized by passing the global phase as argument. Check that the two circuits have the same unitary.

Reply:
A new test has been added in to check the compatibility of the GLobalPhaseGate with the preexisting feature.